lib/omnibus/packagers/base.rb

```diff
@@ -127,6 +127,25 @@ def install_dir
     #
     expose :windows_safe_path
 
+    #
+    # Skip this packager during build process
+    #
+    # @example
+    #   skip_package true
+    #
+    # @param [TrueClass, FalseClass] value
+    #   whether to delay validation or not
+    #
+    # @return [TrueClass, FalseClass]
+    #   whether to skip this packager type or not
+    def skip_packager(val = false)
+      unless val.is_a?(TrueClass) || val.is_a?(FalseClass)
+        raise InvalidValue.new(:iwix_light_delay_validation, "be TrueClass or FalseClass")
```

Comment:
The first parameter to `InvalidValue.new` should be [a symbol of the method name](https://github.com/chef/omnibus/blob/3eefb1cd8de69b1d97de2962f779512892e9296d/lib/omnibus/exceptions.rb#L158-L159) that received the invalid value. So `:iwix_light_delay_validation` should probably be `:skip_packager` here.